src/pages/showcase.js

```diff
@@ -15,102 +15,109 @@ const projects = [
     title: 'GUIJS',
     image: imgPath + 'guijs/logo.svg',
     link: 'https://guijs.dev/',
-    screenshots: ['0.png'].map((image) => imgPath + 'guijs/' + image),
```

Comment:
There's a lot of formatting changes that shouldn't be here, so please remove all those changes. The repo does have prettier configured for formatting.